system/HTTP/URI.php

```diff
@@ -787,51 +775,54 @@ protected function filterPath(string $path=null)
 		$path = $this->removeDotSegments($path);
 
 		// Fix up some leading slash edge cases...
-		if (strpos($orig, './') === 0) $path = '/'. $path;
-		if (strpos($orig, '../') === 0) $path = '/'. $path;
+		if (strpos($orig, './') === 0) $path	 = '/'.$path;
+		if (strpos($orig, '../') === 0) $path	 = '/'.$path;
 
 		// Encode characters
 		$path = preg_replace_callback(
-			'/(?:[^' . self::CHAR_UNRESERVED . ':@&=\+\$,\/;%]+|%(?![A-Fa-f0-9]{2}))/',
-			function (array $matches)
-			{
-				return rawurlencode($matches[0]);
-			},
-			$path
+			'/(?:[^'.self::CHAR_UNRESERVED.':@&=\+\$,\/;%]+|%(?![A-Fa-f0-9]{2}))/', function (array $matches)
+		{
+			return rawurlencode($matches[0]);
+		}, $path
 		);
 
 		return $path;
 	}
 
 	//--------------------------------------------------------------------
 
-
-
 	/**
 	 * Saves our parts from a parse_url call.
 	 *
 	 * @param $parts
 	 */
 	protected function applyParts($parts)
 	{
-		if (! empty($parts['host'])) $this->host = $parts['host'];
-		if (! empty($parts['user'])) $this->user = $parts['user'];
-		if (! empty($parts['path'])) $this->path = $this->filterPath($parts['path']);
-		if (! empty($parts['query'])) $this->query = $this->filterQuery($parts['query']);
-		if (! empty($parts['fragment'])) $this->fragment = $this->filterQuery($parts['fragment']);
+		if ( ! empty($parts['host'])) $this->host		 = $parts['host'];
+		if ( ! empty($parts['user'])) $this->user		 = $parts['user'];
+		if ( ! empty($parts['path'])) $this->path		 = $this->filterPath($parts['path']);
+		if ( ! empty($parts['query']))
+				$this->query	 = $this->filterQuery($parts['query']);
+		if ( ! empty($parts['fragment']))
+				$this->fragment	 = $this->filterQuery($parts['fragment']);
 
```

Comment:
My personal opinion would be that the style guide would have a reference to single line control structures do not need braces, but if split on to separate lines, must have curly braces wrapping it. This would still allow for very simple `if ($foo == $bar) return $baz;` situations, which make sense, while keeping maintenance easier in the long run. 

And yes, getting the settings right in the IDE are a pain. :)